networking/synthetic/scripts/tuning.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+INTERFACE=$1
+
+tuna -c22 -q "{$INTERFACE}" -m -x
+tuned-adm profile stac || exit 1
+
+# additional tuning necessary if using nohz_full:
```

Comment:
this can move to the tuned profile in a [script] section.  see https://github.com/redhat-performance/tuned/tree/master/profiles/realtime for an example of that.